src/components/checkbox/Checkbox.tsx

```diff
@@ -67,10 +70,11 @@ const Checkbox = ({
             <input
                 aria-describedby={description ? `description_${inputID}` : ''}
                 checked={isChecked}
-                className={inputClassName}
+                className={classNames(inputClassName)}
```

Comment:
we can remove `classNames`
```suggestion
                className={inputClassName}
```

Reply:
Oh, right.